MAUI/TextInputLayout/Supported-input-views.md

```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
 {% endhighlight %}
 {% endtabs %}
 
-![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
\ No newline at end of file
+![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
+
+## Picker
+
+To initialize the `Picker` control and launch it in each platform, refer to the `getting started with picker` documentation.
```

Comment:
Add Picker documentation (https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker)

Reply:
Addressed the changes